sjb/config/common/test_cases/origin_release_install_azure.yml

```diff
@@ -0,0 +1,194 @@
+    - type: "script"
+      title: "push the release"
+      repository: "origin"
+      script: |-
+        sudo chmod a+rw /tmp/.docker/config.json
+        DOCKER_CONFIG=/tmp/.docker OS_PUSH_LOCAL=1 OS_PUSH_ALWAYS=1 hack/push-release.sh
+    - type: "script"
+      title: "push the image registry release"
+      repository: "image-registry"
+      script: |-
+        DOCKER_CONFIG=/tmp/.docker OS_PUSH_LOCAL=1 OS_PUSH_ALWAYS=1 hack/push-release.sh
+    - type: "script"
+      title: "push the kubernetes metrics server release"
+      repository: "kubernetes-metrics-server"
+      script: |-
+        DOCKER_CONFIG=/tmp/.docker OS_PUSH_LOCAL=1 OS_PUSH_ALWAYS=1 hack/push-release.sh
+    - type: "script"
+      title: "push the web console release"
+      repository: "origin-web-console-server"
+      script: |-
+        DOCKER_CONFIG=/tmp/.docker OS_PUSH_LOCAL=1 OS_PUSH_ALWAYS=1 hack/push-release.sh
+        sudo rm -rf /tmp/.docker
+
+    - type: "host_script"
+      title: "move secrets to remote host"
+      script: |-
+        rsync --copy-links --omit-dir-times --archive --rsh "ssh -F ./.config/origin-ci-tool/inventory/.ssh_config" /var/lib/jenkins/.azure/* openshiftdevel:/home/origin/release/cluster/test-deploy/${CLUSTER_PROFILE}/
+        ssh -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel 'sudo chown -R origin:origin-git /home/origin/release/cluster/test-deploy/${CLUSTER_PROFILE}/'
+        ssh -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel 'sudo chmod -R ug+rwX /home/origin/release/cluster/test-deploy/${CLUSTER_PROFILE}/'
+
+    - type: "script"
+      title: "provision test cluster"
+      timeout: 2800
+      script: |-
+        sudo systemctl restart docker
+
+        suffix='origin-${component}'
+        suffix="${suffix}:$( cat /data/src/github.com/openshift/aos-cd-jobs/ORIGIN_TAG )"
+        pushd release/cluster/test-deploy/${CLUSTER_PROFILE}
+        TYPE=$CLUSTER_PROFILE ../../bin/ansible.sh ansible-playbook \
+          -e "openshift_test_repo=${location_url}" \
+          -e "debug_level=5" \
+          -e "openshift_azure_base_image_ns=opstest" \
+          -e "openshift_azure_base_image=rhel7.4_base-20180323133134" \
+          -e "openshift_azure_image=${INSTANCE_PREFIX}-openshift-gi" \
+          -e "openshift_azure_clusterid=magentaci${INSTANCE_PREFIX}" \
+          playbooks/${CLUSTER_PROFILE}/openshift-cluster/build_image.yml
+        cp admin.kubeconfig /tmp/cluster-admin.kubeconfig
+        popd
+  # post_actions:
+  #   - type: "script"
+  #     title: "deprovision test cluster"
+  #     repository: "release"
+  #     timeout: 900
+  #     script: |-
+  #       trap 'exit 0' EXIT
+  #       pushd release/cluster/test-deploy/${CLUSTER_PROFILE}
+  #       TYPE=$CLUSTER_PROFILE ../../bin/ansible.sh ansible-playbook \
+  #         -e "openshift_azure_clusterid=magentaci${INSTANCE_PREFIX}" \
+  #         playbooks/${CLUSTER_PROFILE}/openshift-cluster/deprovision.yml
```

Comment:
Seems like @kwoodson's branch has this playbook around. We should do deprovisioning.